test/hummingbot/core/api_throttler/test_api_throttler_base.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from hummingbot.core.api_throttler.api_throttler_base import APIThrottlerBase
+from hummingbot.core.api_throttler.data_types import RateLimit
+
+
+class MockAPIThrottler(APIThrottlerBase):
+
+    def execute_task(self):
+        return id(self)
+
+
+class APIThrotterBaseUnitTests(unittest.TestCase):
```

Comment:
If APIThrottlerBase is going to be an abstract class, it will not make much sense to have tests for it directly (we will only need to have tests for the concrete subclasses)

Reply:
Duly noted.